aiida_common_workflows/workflows/bands/generator.py

```diff
@@ -38,22 +38,21 @@ def define(cls, spec):
         )
         spec.input_namespace(
             'engines',
-            required=False,
             help='Inputs for the quantum engines',
         )
         spec.input_namespace(
             'engines.bands',
-            help='Inputs for the quantum engine performing the calculation of bands.',
+            help='Inputs for the quantum engine performing the bands calculation.',
         )
         spec.input(
             'engines.bands.code',
             valid_type=orm.Code,
             serializer=orm.load_code,
-            help='The code instance to use for the bands calculation.',
+            help='The code instance to use for the geometry optimization.',
```

Comment:
Wait what? The `bands.code` is used for the geometry optimization?

Reply:
No jeez! Sorry! So it was correct. Why did I mess up!